api/users/views.py

```diff
@@ -288,25 +287,23 @@ class UserNodes(JSONAPIBaseView, generics.ListAPIView, UserMixin, ODMFilterMixin
     view_category = 'users'
     view_name = 'user-nodes'
 
+    ordering = ('-date_modified')
```

Comment:
Yes, we are currently filtering v2/nodes/ on -date_modified, so it v2/users/me/nodes/ should probably be filtered the same way.